### DISCUSSION TITLE
DOSBench

### DISCUSSION THREAD

**Discussion by Grounded0** [dosbox-staging/dosbox-staging#3895] [General]
### Download

[dosbench.zip](https://github.com/user-attachments/files/21093233/dosbench.zip) (14 MB)

### Changelog

20 June 2025:

- Reverted CHKCPU to v1.23 due to v1.27.1 crashing when trying to check cputype=486
- Filename is now dosbench.zip (all lower case) to comply with today's norms on filenames

29 August 2024:

- VBETEST (VBE Compliance Test) added
- 3DBench 1.0 and MCPDIAG run with LOADFIX due to both failing to run
after running VBETEST with its PMODE/W DOS extender
- Tweaked text labels in DOSBENCH.BAT

26 August 2024:

- MCPDIAG (FPU test) added
- Tweaked text labels in DOSBENCH.BAT

24 August 2024:

- CHKCPU updated to v1.27.1
- CHKCPU runs in verbose mode for more CPU details
- DOSBENCH.BAT marks CHKCPU as updated
- CHKCPU release notes included in \MARKS\CHKCPU
- More about the release: http://www.steunebrink.info/chkcpu.htm

Prior to 24 August 2024:

- Didn't document changes before this date but at least:
- TOPBENCH has been updated
- SPEEDTST, PERFDOS, Linpack, MMX Speed Test has been added

### Issues

If you notice any problems or issues with the current release, please post a comment below, and I'll take a look.

**Comment by Torinde** [comment]
> Final release issued by @Grounded0

@Grounded0, I hope you'll still be around!

**Reply by Grounded0** [reply]
> @Grounded0, I hope you'll still be around!

I'll still be around in Spain spending money coming from the pension fund and enjoying the company of mamacitas.

**Reply by johnnovak** [reply]
Leisure Suit Grounded0 😎 🤘🏻

**Reply by MasterO2** [reply]
With Grounded0's pension, maybe he's already at La Costa Lotta?   😄

**Comment by Grounded0** [comment]
I've uploaded this directly on GitHub, since GitHub has raised maximum allowed file size for normal users.

**Comment by Grounded0** [comment]
~~- Final release issued by @Grounded0~~

I'll keep supporting this, but only by fixing issues or problems from now on.
